README.md

```diff
@@ -75,6 +75,35 @@ Accept: application/json
 
 The underlining `NSMutableURLRequest` object can be accessed via the `urlRequest` property.
 
+## Testing
+
+### DVR
+JSONRequest uses [DVR](https://github.com/venmo/DVR) for testing. DVR records the HTTP interactions of the tests and replays them during future runthroughs.
+
+#### Usage
+Each test target should contain a setup function that should look like this:
+```swift
+override func setUp() {
+    JSONRequest.requireNetworkAccess = false
+    // anything else that needs to be setup
+    super.setUp()
+}
+```
+Setting the ```requireNetworkAccess``` variable disables errors due to lack of internet, allowing tests to pass wherever you might need to test.
+
+**It should be noted that internet access is REQUIRED the first time a test is run so that DVR can record the responses given.**
+
+After you have your target setup you can test easily by creating your own instance of JSONRequest and pointing it to the stored response file.
+
+```swift
+func testMyAmazingChange() {
+    let jsonRequest = JSONRequest(session: DVR.Session(cassetteName: "testFiles/testMyAmazingChange"))
+    // all network calls and asserts using jsonRequest
+}
+```
+The first time the test is run DVR will record the requests and responses made, storing both in the file indicated.
+When the test has finished DVR will abort testing and print out the location of the saved file which you will need to add to xcode.
+
 ## Why Synchronous? Are you crazy?
```

Comment:
If you were to add the text, "Possibly" underneath this heading... I wouldn't be opposed.